constellations/orion/src/repos/account_repo_test.go

```diff
@@ -106,6 +106,71 @@ func Test_SelectAccountByPrimaryEmail(t *testing.T) {
 	}
 }
 
+//
+// Get accounts with negative balances
+//
+func Test_SelectAllNegativeBalances(t *testing.T) {
+	db, mock, repo := initAccountTest(t)
+	defer db.Close()
+
+	// Mock DB statements and execute
+	rows := sqlmock.NewRows([]string{"Id", "CreatedAt", "UpdatedAt", "DeletedAt", "PrimaryEmail", "Password", "Sum"}).
+		AddRow(
+			1,
+			now,
+			now,
+			sql.NullTime{},
+			"test@gmail.com",
+			"password",
+			-300,
+		).
+		AddRow(
+			2,
+			now,
+			now,
+			sql.NullTime{},
+			"test2@gmail.com",
+			"password2",
+			-200,
+		)
+	mock.ExpectPrepare("^SELECT (.*) FROM accounts (.+)").
```

Comment:
I wasn't really sure how to mock the sql statement and match up the expressions

Reply:
This works, but you could also make it match up more closely if you want. You just need to make sure the regex matches `SELECT accounts.*,SUM(amount) FROM accounts JOIN transactions ON accounts.id=transactions.account_id GROUP BY account_id HAVING SUM(amount) < 0`.